code/modules/mining/laborcamp/laborstacker.dm

```diff
@@ -93,6 +93,8 @@ GLOBAL_LIST(labor_sheet_values)
 	if(..())
 		return
 	var/mob/M = usr
+	if(!istype(M))
```

Comment:
Is this not already checking for if they're a mob that can actually interact with a computer? If it isn't then this needs a more robust check.